### PR TITLE
feat: add/update benchmark of loss prevention pipelines documentation and make targets

### PR DESCRIPTION
**Pull request by jim-wang-intel**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/loss-prevention/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [x] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?

- add benchmark documentation
- update make targets: benchmark, benchmark-stream-density to have the right parameters for running result outputs
- add make target: clean-benchmark-results to be easier for cleaning between new sets of benchmarks

## Issue this PR will close

close: **#15**

## Test Instructions if applicable

- git clone this PR
- follow the instructions documented in[ benchmark.md](https://github.com/jim-wang-intel/loss-prevention/blob/benchmark-pipelines-results/benchmark.md)
- you should be able to run the benchmark and see the results for both number of pipelines and stream density for loss prevention

### CODE REVIEW

**Review by seanohair22 — CHANGES_REQUESTED** [review]
On my run, I had to specify python3 in the make targets and in the benchmark-scripts target `init-packages` I had to force the packages to install with `--break-system-packages` since I am not using a venv...

**Review by antoniomtz — APPROVED** [review]
LGTM. Ran successfully

**Comment by jim-wang-intel** [comment]
> On my run, I had to specify python3 in the make targets and in the benchmark-scripts target `init-packages` I had to force the packages to install with `--break-system-packages` since I am not using a venv...

Thanks @seanohair22 for letting us know: the existing documentation suggests we use vent [in Prerequisite section](https://github.com/intel-retail/documentation/blob/main/docs_src/performance-tools/benchmark.md#prerequisites); but that is a good testing for non-virtual env. Usually I would also suggest running in venv so that if somehow python is messed up, you are less miserable :-) .

File: benchmark.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+## Clean up
+
+To clean up all the benchmark results, run the command:
+
+```bash
+make clean-benchmark-results
+```
+
+This comes handy when you want to have a new set of benchmark for different benchmark use cases like different pipelines or running duration.
```
[the remark this answers is not in the document]
Comment (jim-wang-intel) [reply benchmark.md]: thank Sean, all typos are fixed :-)

**Review by brian-intel — APPROVED** [review]

**Review by seanohair22 — APPROVED** [review]

**Pull request merged**